wunderio/client-UK-FSA-beta
# FSA-789 Skip to content visibility is unreliable with VoiceOver

To test: pull to local and `npm run build` 
Activate VoiceOver and navigate to `skip to main content` button, it should not mess the announcements anymore.
